lib/rest-connector.js

```diff
@@ -32,7 +35,6 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
       for (var i in dataSource.constructor.DataAccessObject) {
         DataAccessObject[i] = dataSource.constructor.DataAccessObject[i];
       }
-      /* eslint-disable one-var */
       for (var i in dataSource.constructor.DataAccessObject.prototype) {
```

Comment:
I think you are misunderstanding the meaning of `one-var`. Read more [here](http://eslint.org/docs/rules/one-var)